packages/lesswrong/server/givingSeason/resolvers.ts

```diff
@@ -9,6 +9,15 @@ import {
 import ElectionVotes from "@/lib/collections/electionVotes/collection";
 import { ACTIVE_DONATION_ELECTION, DONATION_ELECTION_AGE_CUTOFF } from "@/lib/givingSeason";
 import { instantRunoffAllPossibleResults, IRVote } from "@/lib/givingSeason/instantRunoff";
+import { memoizeWithExpiration } from "@/lib/utils/memoizeWithExpiration";
```

Comment:
There are many different hand rolled caching implementations in the codebase, this seems like a perfectly good one